Cargo.toml

```diff
@@ -45,7 +45,7 @@ instrument = ["rftrace", "rftrace-frontend"]
 align-address = "0.3.0"
 byte-unit = { version = "5", features = ["byte"] }
 clap = { version = "4.5", features = ["derive", "env"] }
-nix = { version = "0.30", features = ["mman", "pthread", "signal"] }
+clean-path = "0.2.1"
```

Comment:
If it is no extra effort, it'd be a little nicer to do the resorting in a separate commit.